controlpanel/frontend/views/user.py

```diff
@@ -118,6 +119,23 @@ def post(self, request, *args, **kwargs):
         return HttpResponseRedirect(reverse_lazy("manage-user", kwargs={"pk": user.auth0_id}))
 
 
+class ReinitialiseUser(OIDCLoginRequiredMixin, PermissionRequiredMixin, View):
+    permission_required = "api.add_superuser"
+    http_method_names = ["post"]
+
+    def post(self, request, *args, **kwargs):
+        user = get_object_or_404(User, pk=kwargs["pk"])
+        cluster_user = ClusterUser(user)
+
+        try:
+            cluster_user._init_user()
```

Comment:
I'm wondering if it might be better to actually [call the `create` method on the cluster object](https://github.com/ministryofjustice/analytics-platform-control-panel/blob/main/controlpanel/api/cluster.py#L285).

As that way, if the users role had been removed it would be created, which currently there is no way for us to do via Control Panel. And digging into the code that calls role creation, if it already exists the [exception is caught](https://github.com/ministryofjustice/analytics-platform-control-panel/blob/main/controlpanel/api/aws.py#L468), so would go on to call the `_init_user()` method.